libclc/ptx-nvidiacl/libspirv/group/group_non_uniform.cl

```diff
@@ -34,3 +35,9 @@ _Z29__spirv_GroupNonUniformBallotjb(unsigned flag, bool predicate) {
 
   return res;
 }
+
+_CLC_DEF _CLC_CONVERGENT uint
+_Z37__spirv_GroupNonUniformBallotBitCountN5__spv5Scope4FlagEiDv4_j(
+    uint scope, uint flag, __clc_vec4_uint32_t mask) {
+  return __clc_native_popcount(__nvvm_read_ptx_sreg_lanemask_lt() & mask[0]);
+}
```

Comment:
I'm surprised that `scope` and `flag` aren't used here.

I think this is probably fine given the way that the intrinsic is currently being used, but it might be a good idea to add a comment here that it assumes the scope is `SubGroup` and the group operation is assumed to be `ExclusiveScan`.

Reply:
Good point thanks, I've now added a comment.